components/Typography/src/MDCTypography.m

```diff
@@ -344,6 +356,7 @@ - (BOOL)isLargeForContrastRatios:(UIFont *)font {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wtautological-pointer-compare"
 #pragma clang diagnostic ignored "-Wunreachable-code"
+#pragma clang diagnostic ignored "-Wpartial-availability"
```

Comment:
Should we be clang diagnostic pop this after the call below?

Reply:
Looks like there was one alredy there.

Reply:
yep!